hack/ebs-scale-test/helpers/cluster-setup/scale-driver-values.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default values.yaml for ebs-scale-test installation of aws-ebs-csi-driver
+image:
+  pullPolicy: Always
+controller:
+  logLevel: 7
+  replicaCount: 1
```

Comment:
This is a very pragmatic solution for solving the issue of different controller pods having different leaders for the sidecars - can we add a comment explaining that? its not immediately obvious.